homeassistant/components/zwave/climate.py

```diff
@@ -244,7 +244,9 @@ def hvac_modes(self):
 
         Need to be a subset of HVAC_MODES.
         """
-        return self._hvac_list
+        if self.values.mode:
+            return self._hvac_list
+        return [HVAC_MODE_HEAT]
```

Comment:
I’m not sure we should hard code heat for all zwave devices.  I think you can return an empty list instead and it will fix your issue.